web/admin/src/editors.js

```diff
@@ -9,13 +9,12 @@ export class TextEditor extends PureComponent {
   }
 
   render() {
-    const {title} = this.props
+    const {title, placeholder} = this.props
 
     return (
-      <div>
-        <label>
-          {title} <input type="text" value={this.state.value} onChange={this.onChange} onBlur={this.onBlur} />
-        </label>
+      <div className="contentBox">
```

Comment:
I think we should leave this as a `<label>` so that screen readers can appropriately associate labels with the input fields. We can still style the label text and input w/ CSS as appropriate.